example/.specify/scripts/bash/get-feature-paths.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+eval $(get_feature_paths)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)